desktop_version/src/FileSystemUtils.cpp

```diff
@@ -32,14 +32,20 @@ int mkdir(char* path, int mode)
 #define MAX_PATH PATH_MAX
 #endif
 
+#ifdef _WIN32
```

Comment:
PhysFS provides this in their API, can you use this instead?

Reply:
Okay, good to know. I've changed it to use the existing function